src/main/java/com/garbagemule/MobArena/waves/MACreature.java

```diff
@@ -241,6 +263,12 @@ private static void registerTypeVariants() {
         //
         put("angrybee", "angrybees", "BEE", null);
         put("angrywolf", "angrywolves", "WOLF");
+        put("babypiglin", "babypiglins", "PIGLIN");
+        put("babyzombifiedpiglin", "babyzombifiedpiglins", "ZOMBIFIED_PIGLIN");
+        put("babyhoglin", "babyhoglins", "HOGLIN");
+        put("babyzoglin", "babyzoglins", "ZOGLIN");
```

Comment:
These lines generate the following errors on 1.13:

```
[18:59:16 WARN]: [MobArena] Failed to register monster type 'babypiglin', because its type was not found: PIGLIN
[18:59:16 WARN]: [MobArena] Failed to register monster type 'babyzombifiedpiglin', because its type was not found: ZOMBIFIED_PIGLIN
[18:59:16 WARN]: [MobArena] Failed to register monster type 'babyhoglin', because its type was not found: HOGLIN
[18:59:16 WARN]: [MobArena] Failed to register monster type 'babyzoglin', because its type was not found: ZOGLIN
```

This is a result of the types piglin, hoglin, and zoglin not existing prior to 1.16, so they'll never resolve on earlier versions. It's the same issue as with angry bees, and the solution is to pass `null` as the last type. Check out the comment in the `put()` method that explains it in more detail.

The `babyzombifiedpiglin` is actually already present as a "broken" type further down (and constitutes a branch in the spawn method's switch statement), so it shouldn't need to be registered here as well.

Please re-sort the puts by the first entry so the list stays sorted. It helps with glanceability since the types are a bit all over the place version-wise :)

Reply:
Oh shoot. Will take care of that now